android/app/src/androidTest/java/org/ZingoLabs/Zingo/RustFFITest.kt

```diff
@@ -77,3 +109,57 @@ class CommandsOnLaunch {
         assertThat(addresses[0].receivers.orchard_exists).isEqualTo(true)
     }    
 }
+
+class ExecuteSyncFromSeed {
+    @Test
+    fun executeSyncFromSeed() {
+        val mapper = jacksonObjectMapper()
```

Comment:
`jacksonObjectMapper`  ?    Interesting name.

Reply:
this was the one i chose to parse json ... there is a few options but this looked straight forward to use to me